entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# if lnd enabled, attempt to connect
+if [[ -z "${LND_RPC_AUTHORITY}" ]]
+then ./bin/agora --directory files --http-port $AGORA_PORT --lnd-rpc-authority $LND_RPC_AUTHORITY --lnd-rpc-cert-path .lnd/tls.cert --lnd-rpc-macaroon-path .lnd/invoices.macaroon
```

Comment:
Related to the other comment. I don't think you should reference `.lnd` directly. Rather, use environment variables for `TLS_CERT_PATH` and `MACAROON_PATH`:

```
--lnd-rpc-cert-path $TLS_CERT_PATH --lnd-rpc-macaroon-path $MACAROON_PATH
```

alternatively, use an environment variable for `LND_DIR`:

```
--lnd-rpc-cert-path $LND_DIR/tls.cert --lnd-rpc-macaroon-path $LND_DIR/invoices.macaroon
```

Reply:
And then you can mount the `.lnd` directory as a volume at run time:
```
docker run -p 8080:8080 -v ~/.lnd:/lnd -e LND_DIR=/lnd -e AGORA_PORT=8080 -e LND_RPC_AUTHORITY=10.0.0.20:10009 agora:latest
```